### PR TITLE
 Fix detection of in-place path_locale when RUN_IN_PLACE=0 

### PR DESCRIPTION
**Pull request by sfan5**
<b>don't squash</b>

### CODE REVIEW

File: src/config.h [hunk 1]
```diff
@@ -28,13 +28,3 @@
 		#define BUILD_TYPE "Debug"
 	#endif
 #endif
-
-#define BUILD_INFO \
```
Comment (SmallJoker) [inline src/config.h:32]: 50b2185ced9dc64e054c
#6461

Comment (sfan5) [reply src/config.h:32]: As far as I've understood, this only affects putting a build date there.

If the version changes, the `cmake_config_githash.h` is updated and then `version.cpp` will also be recompiled. Am I missing something?

**Comment by Wuzzy2** [comment]
Works for me. ^^

File: src/config.h (hunk 1 above)
Comment (SmallJoker) [reply src/config.h:32]: Makes sense. We can investigate this if #defines other than the build date are affected.

**Pull request merged**